src/tequila/quantumchemistry/qc_base.py

```diff
@@ -595,16 +595,134 @@ def orthonormalize_basis_orbitals(self):
         # backward compatibility
         return self.use_native_orbitals()
     
-    def use_native_orbitals(self, inplace=False):
+    def use_native_orbitals(self, inplace=False, core: list = None, *args, **kwargs):
         """
+        Parameters
+        ----------
+        inplace: update current molecule or return a new instance
+        core: list of the core orbitals indices (they will be frozen) on the current orbital schema. If not provided, they will be employed the
+        indices on the integral manager. Indices interpreted with respect to the complete basis. If core provided but not active, will be
+        chosen the ones with the lowest overlap on the Native schema.
+        active(in kwargs): list of the active orbital indices on the Native Orbs schema. If not provided they will be chosen the ones
+        with the lowest overlap with the core orbitals. For an example where it may necesary, check the H-He-H molecule
+        with pyscf; the fist HF orbital (at high interatomic distances) correspond to |\phi_0> = 1.|\chi_1>, being identical to the native \varphi_1>.
         Returns
```

Comment:
I think it's  helpful to have that small example here. Good idea. Can you write that comment a bit clearer